web/src/features/auth/AuthWrapper.tsx

```diff
@@ -1,17 +1,27 @@
 import React, { useEffect } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
-
-import { selectAuthentication } from 'app/rootReducer'
-import {
-  authenticate,
-  setAxiosRequestInterceptors
-} from 'features/auth/slices/authenticationSlice'
+import { authenticate } from 'features/auth/slices/authenticationSlice'
+import axios from 'api/axios'
+import { AppThunk } from 'app/store'
+import { selectToken, selectAuthentication } from 'app/rootReducer'
 
 interface Props {
   shouldAuthenticate: boolean
   children: React.ReactElement
 }
 
+export const setAxiosRequestInterceptors = (): AppThunk => (_, getState) => {
```

Comment:
Can't we drop the export now?
```suggestion
const setAxiosRequestInterceptors = (): AppThunk => (_, getState) => {
```